### DISCUSSION TITLE
Engram layout in the real world (photo gallery)

### DISCUSSION THREAD

**Discussion by sunaku** [binarybottle/engram-layout#52] [Show and tell]
I've been enjoying Arno's Engram 2.0 layout for nearly 2 years now on various keyboards, including the following split columnar ergonomic keyboards that feature Engram natively on their keycaps. :tipping_hand_person: So I thought I'd share some pictures here to help promote Engram. :nerd_face:  Cheers.

![Photo of MoErgo Glove80](https://sunaku.github.io/moergo-glove80-keyboard-photograph.jpg)
![Photo of Ergohaven keyboards](https://sunaku.github.io/ergohaven-keyboards-photograph.jpg)
![Photo of Erogohaven Dactyl Manuform 5x6](https://sunaku.github.io/dactyl-manuform-5x6-keyboard-photograph.jpg)
![Photo of Ergohaven Remnant](https://sunaku.github.io/ergohaven-remnant-keyboard-photograph.jpg)

**Comment by binarybottle** [comment]
Delightful-looking!  Thank you for sharing, @sunaku!!

**Comment by sunaku** [comment]
Added [my MoErgo Glove80 keyboard](https://sunaku.github.io/moergo-glove80-keyboard-photograph.html) to the gallery above. :camera_flash: 

![Photo of MoErgo Glove80](https://sunaku.github.io/moergo-glove80-keyboard-photograph.jpg)
![Layout of MoErgo Glove80](https://sunaku.github.io/moergo-glove80-keyboard-base-layer.png)
